dyn/tm/accounts.py

```diff
@@ -750,6 +764,42 @@ def delete_permissions_group(self, group):
         uri = '/UserGroupEntry/{}/{}/'.format(self._user_name, group)
         DynectSession.get_session().execute(uri, 'DELETE')
 
+    def add_zone(self, zone):
+        """Add individual zones to this :class:`~dyn.tm.accounts.User`
+
+        :param zone: the zone to add
+        """
+        if zone not in self._zone:
+            self._zone.append(zone)
+            uri = '/UserZoneEntry/{}/{}/'.format(self._user_name, zone)
+            DynectSession.get_session().execute(uri, 'POST')
+
+    def replace_zone(self, zones):
```

Comment:
Doesn't look like this actually sends anything to the API. Also, docstring might not be right? (or at least, its unclear)...